docs/guide/standalone-search.md

```diff
@@ -0,0 +1,42 @@
+---
+outline: deep
+---
+
+# Using GeoNetwork-UI Standalone Search
+
+GeoNetwork-UI offers a mechanism similar to Web Components but which only allows you to search in a GeoNetwork catalog using its search interface.
+
+The JS file to include is as a result much lighter, and this system is entirely appropriate if the search results coming from GeoNetwork should be used in a different UI framework.
+
+## Use
+
+To use the Standalone Search, you have to:
+
+- import the Standalone Search script exported by Angular (available via jsdelivr)
+- call `GNUI.init(...)` with the proper GeoNetwork API URL.
```

Comment:
Wondering if it's necessary to point out in the doc that it could potentially create a conflict with another global var?

Reply:
I don't think this is likely; on the other hand, having a conflict detection logic could make sense, I'll add that :)